app/filters/autolink_filter.rb

```diff
@@ -13,8 +13,12 @@ def process(post)
   private
 
   def autolink(url)
-    if url =~ /\.(jpg|jpeg|gif|png)$/i
+    if url =~ /imgur.*\.(gif)$/i
+      "<video loop controls autoplay><source src=\"" + url.sub(/\.gif/, '.mp4') + "\" type=\"video/mp4\"></video>"
```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.